### ISSUE TITLE
Adding icons on the navbar 

### ISSUE THREAD

**Issue by Soumya6Tiwari** [itsmohit097/medi-hub#226]
Adding icons on the navbar for different sections such as doctors , appointments etc .

**Comment by Soumya6Tiwari** [comment]
@itsmohit097  I want to work on this issue . Kindly assign me

**Comment by ishikat** [comment]
Hii @itsmohit097 
I would like to add icons to the navbar.
can you assign it to me under GSSOC24.

**Comment by itsmohit097** [comment]
@Soumya6Tiwari good idea. Im making some changes to the structure of the project. After that I'll assign u. Atleast wait for 2-3 days🙏

**Comment by Soumya6Tiwari** [comment]
@itsmohit097  sure :)
